bin/run-all-fuzzers.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+# A utility script to run all fuzzers on IC for limited turns defined under MAX_EXECUTIONS
+set -ex
+
+MAX_EXECUTIONS=100
+if [[ -n "$2" ]]; then
+    MAX_EXECUTIONS=$2
+fi
+
+case $1 in
+    -h | --help)
+        cat <<EOF >&2
+        usage:
+            $0 --libfuzzer 100 # Run all libfuzzer targets. Each target will have 100 executions.
+            $0 --afl 100       # Run all afl targets. Each target will have 100 executions.
+EOF
+        exit 0
+        ;;
+    --libfuzzer)
+        bazel build --config=fuzzing --build_tag_filters=libfuzzer //rs/...
+        LIST_OF_FUZZERS=$(bazel query 'attr(tags, "libfuzzer", //rs/...) except attr(tags, "sandbox_libfuzzer", //rs/...)')
+        for FUZZER in $LIST_OF_FUZZERS; do
+            bazel run --config=fuzzing $FUZZER -- -runs=$MAX_EXECUTIONS
```

Comment:
Since this is just to check that the fuzzers are working correctly, would it make sense to limit the options of libfuzzers? (i.e. -max_len, -timeout, etc)? Or do you want to have an execution as close as possible to the one in prod?

(same thing with afl)

Reply:
I will add them later on if we see slowdowns. The CI job already has a global timeout and AFL already enforces timeout in the script.